exercises/grade-school/test/test_grade_school.c

```diff
@@ -99,27 +107,17 @@ static void test_adding_students_to_different_grades_adds_to_same_roster(void)
    check_roster(&input, &expected);
 }
 
-static void test_students_in_same_grade_are_sorted_in_same_roster(void)
+static void
+test_grade_returns_an_empty_list_if_there_are_no_students_enrolled(void)
 {
    TEST_IGNORE();
-   roster_t input = {
-      3, {
-          (student_t) {1, "Peter"},
-          (student_t) {1, "Anna"},
-          (student_t) {1, "Barb"}}
-   };
-
-   roster_t expected = {
-      3, {
-          (student_t) {1, "Anna"},
-          (student_t) {1, "Barb"},
-          (student_t) {1, "Peter"}}
-   };
+   roster_t actual = get_roster();
```

Comment:
This should use `get_grade`, right?

Reply:
Typo by me here, the test is intended to check an empty roster.
This makes your last comment also.